src/views/portals/YetiView.vue

```diff
@@ -566,6 +579,21 @@
 
     mounted() {
       this.check();
+      const doc = this.$route.params.document;
+      const lang = this.$language.current;
+      if (doc) {
+        c2c['route'].getCooked(doc, lang).then(doc => {
```

Comment:
Can you have a look on other place where we load data ? Basically, we use something like : 

```
This.promise = load() ..

properties {
  data() {
    return this.promise.data
}
``` 

And you have a `<loading notification> component` that display errors.